nautobot/core/tests/runner.py

```diff
@@ -30,3 +40,87 @@ def __init__(self, **kwargs):
             kwargs["exclude_tags"] = incoming_exclude_tags
 
         super().__init__(**kwargs)
+
+    def generate_report(self, test_results, result):
+        """
+        Generate Performance Report consists of unittests that are significantly slower than baseline.
+        """
+        test_result_count = len(test_results)
+
+        # Add `--generate_report` to the end of `invoke` commands to generate a report.json file consist of the performance tests result
+        if self.report_path:
+            data = {
+                "tests": [{"name": func_name, "execution_time": float(timing)} for func_name, timing in test_results],
```

Comment:
Is there any intrinsic ordering to `test_results`? If not, can we sort it in some fashion? Thinking about keeping the diffs readable as and when we need to regenerate this file.

(edit) Ah I see, currently it's sorted by execution time? I think sorting by name might be better?